config/locales/fil/devise.fil.yml

```diff
@@ -0,0 +1,59 @@
+# Mga karagdagang mga pagsasalin ng salita ay makikita sa https://github.com/plataformatec/devise/wiki/I18n
+
+fil:
+  devise:
+    confirmations:
+      confirmed: "Ang iyong account ay matagumpay na nakomperma."
```

Comment:
Online dictionaries are saying this might be nakompirma instead?

nakomperma --> nakumpirma